interface-definitions/syslog.xml

```diff
@@ -0,0 +1,859 @@
+                </valueHelp>
+                <valueHelp>
+                  <format>local5</format>
+                  <description>Local facility 5</description>
+                </valueHelp>
+                <valueHelp>
+                  <format>local6</format>
+                  <description>Local facility 6</description>
+                </valueHelp>
+                <valueHelp>
+                  <format>local7</format>
+                  <description>Local facility 7</description>
+                </valueHelp>
+              </properties>
+              <children>
+                <leafNode name="level">
+                  <properties>
+                    <help>Logging level</help>
+                    <completionHelp>
+                      <list>emerg alert crit err warning notice info debug all</list>
+                    </completionHelp>
+                    <valueHelp>
+                      <format>emerg</format>
+                      <description>Emergency messages</description>
+                    </valueHelp>
+                    <valueHelp>
+                      <format>alert</format>
+                      <description>Urgent messages</description>
+                    </valueHelp>
+                    <valueHelp>
+                      <format>crit</format>
+                      <description>Critical messages</description>
+                    </valueHelp>
+                    <valueHelp>
+                      <format>err</format>
+                      <description>Error messages</description>
+                    </valueHelp>
+                    <valueHelp>
+                      <format>warning</format>
+                      <description>Warning messages</description>
+                    </valueHelp>
+                    <valueHelp>
+                      <format>notice</format>
+                      <description>Messages for further investigation</description>
+                    </valueHelp>
+                    <valueHelp>
+                      <format>info</format>
+                      <description>Informational messages</description>
+                    </valueHelp>
+                    <valueHelp>
+                      <format>debug</format>
+                      <description>Debug messages</description>
+                    </valueHelp>
+                    <valueHelp>
+                      <format>all</format>
+                      <description>Log everything</description>
+                    </valueHelp>
+                  </properties>
+                </leafNode>
+              </children>
+            </tagNode>
+          </children>
+        </tagNode>
+        <tagNode name="host">
+          <properties>
+            <help>Logging to a remote host</help>
+            <valueHelp>
+              <format>x.x.x.x or host.domain.tld</format>
+              <description>Remote host name or IP address</description>
+              <!-- regex host or IP check? -->
```

Comment:
Perhaps no check at all — FQDNs are notoriously hard to check...

Reply:
If unchecked a 'set system syslog host aaa bbb ...' inserts you a space, that leads to a lot of fun in the parser. We definitely need to check it, or I need to extend syslog.py and check it there for whites spaces, backslashes etc. Just wanted to know your opinion and how you guys usually deal with that.

Reply:
set system syslog host "host name" facility all

+        host host name {
+            facility all {
+            }
+        }


would be the result, I can check in verify() but it will be already in the config then, that's why I think checking it with a regex is better.